### PR TITLE
systemvmtemplate: update to Debian 10.9

### PR DESCRIPTION
**Pull request by rhtyd**
Updates systemvmtemplate build scripts to use latest 10.9 iso. Fix url to get `vhd-util`.

### CODE REVIEW

**Review by weizhouapache — APPROVED** [review]
tested ok

**Comment by weizhouapache** [comment]
@rhtyd 
by the way, would it be better to add "--no-check-certificate " to all "wget" command ?

**Comment by rhtyd** [comment]
Not necesary if we install the latest ca-certificates, we don't want to install anything with an invalid certificate.

File: tools/appliance/systemvmtemplate/template.json [hunk 1]
```diff
@@ -36,8 +36,8 @@
       "disk_size": 2500,
       "disk_interface": "virtio",
       "net_device": "virtio-net",
-      "iso_url": "https://cdimage.debian.org/debian-cd/10.8.0/amd64/iso-cd/debian-10.8.0-amd64-netinst.iso",
-      "iso_checksum": "934336d266535c91fcd12cd122c81f8261721efa117fdcb9a31615caa96c5c5ce3454ed5f28f1b25a7b1b5f44631fdfa78a93adb6445e2e2caaf6455ab344cf8",
+      "iso_url": "https://cdimage.debian.org/debian-cd/10.9.0/amd64/iso-cd/debian-10.9.0-amd64-netinst.iso",
+      "iso_checksum": "47d35187b4903e803209959434fb8b65ead3ad2a8f007eef1c3d3284f356ab9955aa7e15e24cb7af6a3859aa66837f5fa2e7441f936496ea447904f7dddfdc20",
```
Comment (sureshanaparti) [inline tools/appliance/systemvmtemplate/template.json:40]: Verified downloads, and checksum. LGTM.

**Review by sureshanaparti — APPROVED** [review]

**Pull request merged**